content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -498,6 +593,14 @@ The Account Resource supports proprietary systems for:
 
 * Account.identifier.system is [Account Number](#account-number) for financial accounts.
 
+### Medications
+
+<h4>MedicationRequest</h4>
+
+The MedicationRequest Resource supports proprietary systems for:
+
+* MedicationRequest.category is [Medication Request Category](#medication-request-category) for medication orders.
```

Comment:
nit: for medication requests

Reply:
https://github.com/cerner/fhir.cerner.com/pull/332/commits/7bee2124b41c1252ae9347951199fbd8db627be3#diff-6fdcb00b5b1b6edd51b687794bea6b95R602